restservice/src/main/java/org/itevents/util/OneTimePassword/OtpGenerator.java

```diff
@@ -0,0 +1,55 @@
+package org.itevents.util.OneTimePassword;
+
+import org.springframework.stereotype.Component;
+
+import java.util.Date;
+import java.util.UUID;
+
+@Component
+ public class OtpGenerator {
+
+    private String password;
+    private Date creationDate;
```

Comment:
Why do we need information about OTP creation date? We need only expiration date. So, this field is redundant.